Makefile

```diff
@@ -18,7 +18,8 @@ create_mix_project:
 # Y - in response to Do you want to publish a Github Wiki for this project? You'd need to manually create the first Github Wiki Page and set the GH_TOKEN and GH_EMAIL secret for this to properly function.
 # Y - in response to Would you like to add the Oban addon?
 # Y - in response to Would you like to add the ExVCR addon?
-common_addon_prompts = Y\nY\nY\nY\nY\nY\nY\nY\n
+# Y - in response to Would you like to add the Gettext Extract and Merge command addon?
```

Comment:
Why not make this added by default i.e., so not an addon? All projects need to be localized and even if there is only one language, the tasks must be there.

Reply:
Yup, that's also a good idea, as i see the default generated Phoenix project is included the Gettext and use it from the start.

![Screen Shot 2022-02-18 at 3 28 20 PM](https://user-images.githubusercontent.com/948856/154646144-f4c4b226-5599-4798-b544-7618980a8e99.png)